lib/StaticNarrative/exporter/data_exporter.py

```diff
@@ -98,12 +89,28 @@ def export_narrative_data(
     return output_data
 
 
-def _reshape_obj(obj_info: list[str | dict[str, Any]], obj_upa: str) -> list[str | dict[str, Any]]:
+def _index_obj(
+    item: dict[str, Any], type_info: dict[str, Any], indexed_data: dict[str, Any]
+) -> None:
+    """Index by object ID and save the type information."""
```

Comment:
May want to note for posterity that this has the side effects of updating the input dicts.